src/structures/votable.ts

```diff
@@ -0,0 +1,65 @@
+import { UserError } from "../errors.ts";
```

Comment:
Already discussed this in Discord; it doesn't seem like there is any reason to include this.

Reply:
For me it is, I ran into the same issue earlier where the returned value was `undefined`. It might be an issue with bun or SQLite since I don't have this behaviour in other projects with drizzle.

Reply:
This seems to still be there. The Votable class can be entirely removed with minimal code changes (the only case where `Suggestion` uses its methods seem to be to update the votes map, which is never actually used by `Suggestion`)